internal/controller/backup-service/reconciler.go

```diff
@@ -690,6 +716,18 @@ func (r *SingleBackupServiceReconciler) waitForDeploymentToBeReady() error {
 
 	if err := wait.PollUntilContextTimeout(context.TODO(),
 		podStatusRetryInterval, podStatusTimeout, true, func(ctx context.Context) (done bool, err error) {
+			deployment, err := r.getBackupSvcDeployment()
+			if err != nil {
+				return false, err
+			}
+
+			// This check is for the condition when deployment rollout is yet to begin, and
+			// pods with new spec are yet to be created.
+			if deployment.Generation > deployment.Status.ObservedGeneration {
+				r.Log.Info("Waiting for deployment to be ready")
```

Comment:
In some places we are logging name of deployment, backup service, configmap but in some places we are not.